src/ProjectionMapping/ProjectionDeckMap/layers.js

```diff
@@ -49,14 +61,15 @@ export const createGeoJsonLayer = (i, layer, GEOGRID) =>
     },
   });
 
+const isFreeMapEnabled = new URLSearchParams(window.location.search).get("usefreemap") === "true";
```

Comment:
could you open a separate PR for this change? It’s easier to test and maintain if we integrate one feature at a time. It looks interesting though, thanks!